src/components/Root/FXHR.test.js

```diff
@@ -70,7 +70,7 @@ describe('FXHR', () => {
     expect(rtr.mock.calls).toHaveLength(0);
   });
 
-  it('If AT is invalid, but RT is valid, refresh the token before sending...', async () => {
+  it.skip('If AT is invalid, but RT is valid, refresh the token before sending...', async () => {
```

Comment:
Are these still valid tests that need to be fixed?

Reply:
Nah, they can just be removed. They relate to inspecting the AT's TTL when making a regular API request and that no longer happens.